distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -73,7 +73,7 @@ config_from_env =
                 IO.println "Unable to determine root project path. JUnit output disabled."
                 Nothing
             False ->
-                (File.new junit_folder) / project_root.name / "JUnit.xml"
+                (File.new junit_folder) / "stdlib" / project_root.name / "JUnit.xml"
```

Comment:
I don't like this addition.

`Standard.Test` is (_meant to be_) a general purpose library for writing tests. These tests currently test our stdlib, but they can be used for any other libraries. So I don't think this belongs here.

Can we restructure how the `ENSO_TEST_JUNIT_DIR` env var is used in the CI? Maybe the engine tests should use a separate env var for setting this path altogether?
Alternatively, why not make them live next to each other, just like `Standard.Base` has directory `Base`, the engine tests could have directory `engine`.

Reply:
I see. Maybe I can try to find a workaround.